aws-rds-dbparametergroup/src/main/java/software/amazon/rds/dbparametergroup/CallbackContext.java

```diff
@@ -30,4 +40,24 @@ public boolean isAddTagsComplete() {
     public void setAddTagsComplete(final boolean addTagsComplete) {
         taggingContext.setAddTagsComplete(addTagsComplete);
     }
+
+    public void timestamp(final String label, final Instant instant) {
+        timestamps.put(label, instant.getEpochSecond());
+    }
+
+    public void timestampOnce(final String label, final Instant instant) {
+        timestamps.computeIfAbsent(label, s -> instant.getEpochSecond());
+    }
+
+    public Instant getTimestamp(final String label) {
+        if (timestamps.containsKey(label) && label != null) {
```

Comment:
if (timestamps.containsKey(label) && label != null) {

Reply:
Or maybe even `timestamps.contains(label)`

Reply:
I don't think we need this as there's always labels provided. I'll remove it and check if there's potential challenges that may arise from having that check. I think this